rpc.go

```diff
@@ -780,7 +780,16 @@ func (c *client) metaLookupForTable(ctx context.Context,
 
 		reg, addr, err := region.ParseRegionInfo(resp)
 		if err != nil {
-			return nil, err
+			// Ignore error, but log if it's anything else than OfflineRegionError. This really
+			// shouldn't happen unless HBase meta table is corrupted/changed format.
+			if _, ok := err.(region.OfflineRegionError); !ok {
+				c.logger.Debug("failed to parse region", "err", err)
+			}
+			continue
+		}
+		if addr == "" {
+			// Ignore region that aren't open/assigned to a RegionServer.
+			continue
 		}
```

Comment:
I think we are already doing this check in `ParseRegionInfo`
https://github.com/tsuna/gohbase/blob/ea1d2bc61069aff06b0eb4aa7f449e99c0f5c225/region/info.go#L150

Reply:
How did I miss that lol